File: Python3/Length of Last Word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        n = len(s)
        count = 0

        for i in range(n-1,-1,-1):
            if count==0 and s[i] == ' ':
                continue
            else:
                if s[i] == ' ':
                    break
            count = count + 1
        return count

File: Golang/Length of Last Word.go
func lengthOfLastWord(s string) int {
    
    n := len(s)
    count := 0

    for i:=n-1;i>=0;i--{
        if count == 0 && s[i] == ' ' {
            continue
        } else {
            if s[i] == ' '{
                break
            }
        }
        count++
    }
    return count
}